src/database.json

```diff
@@ -708,12 +733,6 @@
     "repo": "nanoid",
     "desc": "A tiny, secure, URL-friendly, unique string ID generator for JavaScript."
   },
-  "nessie": {
-    "type": "github",
-    "owner": "halvardssm",
-    "repo": "deno-nessie",
-    "desc": "A database migration tool for deno."
-  },
```

Comment:
Why is this being removed?

Reply:
ohhh... this was not my intension... will fix imediately

Reply:
should, be back...
sorry, I really don't know why this happened